src/extension.ts

```diff
@@ -94,10 +111,27 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 	return Promise.resolve(context);
 }
 
+export async function _stopExistingDfaRun(context: vscode.ExtensionContext, outputChannel: vscode.LogOutputChannel) {
+	const pid = context.workspaceState.get(Constants.GLOBAL_DFA_PROCESS);
```

Comment:
nitpick. Now that it is a workspace setting and not a global setting. Do we want the name of the constant to still be GLOBAL_DFA_PROCESS?

Reply:
Will take care of this.